drivers/serial/uart_sifive.c

```diff
@@ -381,17 +381,17 @@ static void uart_sifive_irq_cfg_func_0(void);
 #endif
 
 static const struct uart_sifive_device_config uart_sifive_dev_cfg_0 = {
-	.port         = DT_SIFIVE_UART0_0_BASE_ADDRESS,
-	.sys_clk_freq = DT_SIFIVE_UART0_0_CLOCK_FREQUENCY,
-	.baud_rate    = DT_SIFIVE_UART0_0_CURRENT_SPEED,
+	.port         = DT_INST_0_SIFIVE_UART0_BASE_ADDRESS,
```

Comment:
@nategraff-sifive @galak Why is the compat named `UART0`? does it make sense that there is a `0` there?

Reply:
Fair question, its how @nategraff-sifive and the SiFive set their compat's.  Not directly related to this PR, but worth asking.

Reply:
Yeah, it's just how they are. The idea is that the 0 gets revved to a 1 if/when the interface changes.